### DISCUSSION TITLE
How to insert custom HTML

### DISCUSSION THREAD

**Discussion by codingmachine16** [GrapesJS/grapesjs#4086] [Q&A]
Hi @artf 

   How can we insert custom html using grapes js. We we import HTML section into plugin, it erases exiting sections. 

Please guide me. Thanks

**Comment by artf** [comment]
You can use [`editor.addComponents`](https://grapesjs.com/docs/api/editor.html#addcomponents).

**Comment by codingmachine16** [comment]
Sir, I want to know is there any plugin or built in block for this purpose?
